.gitignore

```diff
@@ -4,6 +4,9 @@
 # tmp
 **/tmp/
 
+# env
+/.env
```

Comment:
Where this file is used?

Reply:
I used it locally for SECRET_KEY_BASE but not sure if it's correct. Will have to check on this when it gets deployed during deployment.